Source/DafnyLanguageServer/Workspace/DocumentObserver.cs

```diff
@@ -42,19 +38,15 @@ class DocumentObserver : IObserver<DafnyDocument> {
       return;
     }
 
-    var previousDiagnostics = LastPublishedDocument.LoadCanceled
-      ? new Diagnostic[] { }
-      : LastPublishedDocument.ParseAndResolutionDiagnostics;
     var internalErrorDiagnostic = new Diagnostic {
       Message =
         "Dafny encountered an internal error. Please report it at <https://github.com/dafny-lang/dafny/issues>.\n" +
         exception,
       Severity = DiagnosticSeverity.Error,
-      Range = LastPublishedDocument.Program.GetFirstTopLevelToken().GetLspRange()
+      Range = new Range(0, 0, 0, 1)
```

Comment:
Hopefully this works. Have you tested it?

Reply:
Tested on VSCode (after your comment), and it works. I added a unit test to verify this stays 0,0,0,1.

I've also tested this on VSCode with master, that emits a -1,-1,-1,-1 range, and it does _not_ work.